firmware/Core/Src/littlefs/flash_driver.c

```diff
@@ -290,7 +301,10 @@ FLASH_error_enum_t FLASH_erase(SPI_HandleTypeDef *hspi, uint8_t chip_number, lfs
 
         if ((status_reg_buffer[0] & FLASH_SR1_ERASE_ERROR_MASK) > 0) {
             // Flash module returned "erase error" via the status register.
-            DEBUG_uart_print_str("Flash erase error\n");
+            LOG_message(
```

Comment:
Shouldn't errors be reported as a `LOG_SEVERITY_WARNING` or ERROR?

Reply:
Good Catch! Must have missed it!